features/base/tracks/actionTypes.js

```diff
@@ -1,3 +1,4 @@
 export const TRACK_ADDED = 'TRACK_ADDED';
+export const TRACK_MUTE_CHANGED = 'TRACK_MUTE_CHANGED';
```

Comment:
As for the name of variable is pretty self descriptive. I don't see much sense in adding a comment like: "This is action name for case when track muted state is changed".